orange/src/test/java/com/hartwig/hmftools/orange/algo/purple/PurpleVariantFactoryTest.java

```diff
@@ -119,4 +123,85 @@ public void testCanCreateVariantFromContext()
         assertEquals(1, purpleVariant.subclonalLikelihood(), 0);
         assertEquals(List.of(1, 2, 3), purpleVariant.localPhaseSets());
     }
+
+    @Test
+    public void testDetermineReportableTranscripts()
```

Comment:
I feel you can slim down this test (and use `TestPurpleVariantFactory` builders). This test currently tests way more than just the reportable transcripts.

Reply:
Simplified the tests, thanks.